packages/block-library/src/image/style.scss

```diff
@@ -207,7 +207,7 @@
 
 	.lightbox-image-container {
 		position: absolute;
-		overflow: hidden;
+		// overflow: hidden;
```

Comment:
I believe, this is no longer required and additionally, it preserves effects such as shadow in the lightbox. But not quite sure if that should be the intended behavior.

| Before | After |
| -- | -- |
| <img width="552" alt="image" src="https://github.com/WordPress/gutenberg/assets/1935113/a9fe0b56-ab9a-46d4-a6b4-80b812682791"> | <img width="552" alt="image" src="https://github.com/WordPress/gutenberg/assets/1935113/2c85d769-74c6-4c4a-8818-99e7f3f3a77e"> |